README.rst

```diff
@@ -127,6 +127,40 @@ To try out changes before applying them, use the ``--dry-run`` option. For insta
 
    ./manage.py lms forum_delete_course_from_mongodb all --dry-run
 
+Building MongoDB Indexes
+------------------------
+
+To optimize MongoDB query performance, it is crucial to create database indexes. The command will create or update indexes and skip them if they already exist.
+
+To create or update MongoDB indexes, execute the following command:
```

Comment:
Commands must be prefixed by "::"

    To create or update MongoDB indexes, execute the following command::

(same below)

Reply:
Yes, I'm updating form the GitHub UI.